### ISSUE TITLE
Support ApplicationInsights connectionstring

### ISSUE THREAD

**Issue by fgheysels** [arcus-azure/arcus.observability#354]
The Arcus sink that allows setting up logging to ApplicationInsights only supports setting up logging through an instrumentation-key.

However, it's been a while already that Application Insights now also [supports connection-strings](https://docs.microsoft.com/en-us/azure/azure-monitor/app/sdk-connection-string?tabs=net).  

We should provide the necessary extension-methods to support setting up Application Insights logging via a connection-string as well.

**Comment by fgheysels** [comment]
InstrumentationKey support will end in 2025

https://azure.microsoft.com/en-us/updates/technical-support-for-instrumentation-key-based-global-ingestion-in-application-insights-will-end-on-31-march-2025/

**Comment by stijnmoreels** [comment]
We will have to check with Serilog if they already support the connection-string variant.

**Comment by fgheysels** [comment]
Wouldn't that make things more complicated ?
Why not just provide an extension method which allows you to pass in the AppInsights connectionstring ?

Difficulty here is: how shall we do that ?   Do we provide a new extension method , as I don't immediately see a way on providing an overload to our existing extension method.

**Comment by stijnmoreels** [comment]
Yeah, the problem is that it would conflict with our current one. So, I was also looking into that, and maybe the `TelemetryClient` could help, as Serilog solves this problem also this way. Users could configure the client more, but yes, absolutely, it will make it more complex.

**Comment by stijnmoreels** [comment]
Any comment on the proposal, @fgheysels ?
